src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/service/MappingService.kt

```diff
@@ -59,6 +59,8 @@ class MappingService(
     roomIdRepository.findOneByPrisonIdAndNomisRoomDescription(prisonId, nomisRoomDescription)
       .map { RoomMappingDto(it.vsipId, it.nomisRoomDescription, it.prisonId, it.isOpen) }
       .switchIfEmpty(Mono.error(NotFoundException(("prison id=$prisonId, nomis room id=$nomisRoomDescription"))))
+
+  fun deleteVisitMappings(): Mono<Void> = visitIdRepository.deleteAll()
```

Comment:
should really have a `trackEvent` here too for, similar to the creation.  Would be good to make the events unique, e.g. `HNVM` + event, so existing event should really be renamed to `HNVM-visit-mapping-created` or something...

Reply:
i did discussed this earlier and came to the conclusion that the request logging was good enough - can add if thats the concensus?